.github/workflows/verify-bundled-files.yml

```diff
@@ -0,0 +1,40 @@
+name: Verify Bundled
+
+on:
+  push:
+  pull_request:
+  schedule:
+   - cron: "0 1 * * *"
+  workflow_dispatch: ~
+
+permissions:
+  contents: read
+
+jobs:
+  VERIFY_BUNDLED_FILES:
+    name: Verify Bundled Files
+    runs-on: ubuntu-22.04
+    steps:
+      - name: git checkout
+        uses: actions/checkout@v5
+
+      - uses: dorny/paths-filter@v3
+        id: changes
+        with:
+          base: master
+          filters: |
+            opcache_jit_ir:
+              - 'ext/opcache/jit/ir/**'
+
+      - name: Opcache JIT IR - Download
+        if: (success() || failure()) && (steps.changes.outputs.opcache_jit_ir == 'true' || (github.event_name != 'push' && github.event_name != 'pull_request'))
```

Comment:
note: `success() || failure()` is here to run even if previous step failed (to always output all failed bundles once more bundles are added)

Reply:
This should be wrapped in `${{ … }}` and `success() || failure()` should be `!canceled()`.

Reply:
done